README.md

```diff
@@ -19,17 +19,16 @@ To run the MAAP API locally using PyCharm, create a Python Configuration with th
 
 ## II. Local development using python virtualenv
 
-**Pre-requisites:**
+**Prerequisites:**
 
 * postgresql
   * Linux: `sudo apt-get install postgresql python-psycopy2 libpq-dev`
   * Mac OSx: `brew install postgresql`
 * pip, python3.7 and virtualenv
 
 ```bash
-python3 -m venv maap-api-nasa # or whatever environment name you choose
+python3 -m venv venv
```

Comment:
I changed this to reduce the likelihood that someone will create a virtualenv in a directory _other_ than the 2 directories already listed in `.gitignore` (`venv` and `venv2`), and accidentally commit it.